8.0/s2i/bin/run

```diff
@@ -54,12 +54,15 @@ else
   echo "-> Cgroups memory limit is set, using HTTPD_MAX_REQUEST_WORKERS=${HTTPD_MAX_REQUEST_WORKERS}"
 fi
 
-if [ ! -v PHP_FPM_LOG_PATH ]; then
-  /bin/ln -s /dev/stderr ${PHP_FPM_LOG_PATH}/error.log
-fi
 
-chmod -R a+rwx ${PHP_FPM_LOG_PATH}
-chown -R 1001:0 ${PHP_FPM_LOG_PATH}
+
+if [ "x$PLATFORM" == "xel9" ] || [ "x$PLATFORM" == "xfedora" ]; then
+  if [ ! -v PHP_FPM_LOG_PATH ]; then
```

Comment:
What is expected here ?
* don't know about -v test
* should probably be ${PHP_FPM_LOG_PATH}

Reply:
See a check for the presence of variable here: https://www.cyberciti.biz/faq/see-check-if-bash-variable-defined-in-script-on-linux-unix-macos/ . Chapter `Checking if a variable is set in Bash or not`

Reply:
The -v VAR returns true if the shell variable VAR is set:
```
[[ -v ${var} ]] && echo "\${var} not defined" 
[[ -v ${var} ]] && echo "\${var} not defined" || echo "\${var} is defined"
[ -v ${var} ] && echo "\${var} not defined" || echo "\${var} is defined"
```

Reply:
I mean -v is not in "test" man page ;)   (and I have never used it, despite ~30y of shell ;)

BTW in all cases, `[ ! -v PHP_FPM_LOG_PATH ]` is wrong, missing **$**

Reply:
I would argue to use `-z` or `-n` here. Afaik `-v` only checks if the variable is set, but I do not think we want to continue in case it is empty.

Reply:
I also find the use of `[ ! -v ]` here weird - we check if the variable is not set and then we use it for a symlink path? Am I missing something crucial?

Reply:
I have resolved it by commit https://github.com/sclorg/s2i-php-container/pull/365/commits/0b18c2a56d66d034ef8edfdc4c45bfc7b502de68